CoolCurrencyWatchApp WatchKit Extension/ExchangeRateRowController.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  RowController.swift
+//  CoolCurrencyWatchApp WatchKit Extension
+//
+//  Created by Vincent Moyo on 2021/11/29.
+//
+
+import UIKit
+import WatchKit
+
+class ExchangeRateRowController: NSObject {
+    @IBOutlet var currencyCodeLabel: WKInterfaceLabel!
```

Comment:
Can these outlets not be made private?

Reply:
I've and they become inaccessible when I implement them in the InterfaceController